server/services/publication-service.js

```diff
@@ -55,23 +49,26 @@ module.exports = ({ strapi }) => ({
 		// handle single content type, id is always 1
 		const entityId = record.entityId || 1;
 
-		const entity = await strapi.entityService.findOne(record.entitySlug, entityId);
+			const entity = await strapi.documents(record.entitySlug).findOne({
+				documentId: record.entityId,
+			});
 
 		// ensure entity exists before attempting mutations.
 		if (!entity) {
 			return;
 		}
 
-		// ensure entity is in correct publication status
-		if (!entity.publishedAt && mode === 'publish') {
```

Comment:
Should we preserve this check? To make sure we don't publish an already published entity?